### PR TITLE
Male zmiany

### PR DESCRIPTION
**Pull request by s19487**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -55,6 +55,11 @@
             <artifactId>logback-classic</artifactId>
             <version>${logback.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.deeplearning4j</groupId>
+            <artifactId>deeplearning4j-nn</artifactId>
```
Comment (MindTheSynapticGap) [inline pom.xml]: What's the purpose of this? Please delete if unnecessary.

File: src/main/java/Network.java [hunk 2]
```diff
@@ -17,12 +20,19 @@ public void start() {
     }
 
     private void performLearning() {
-        final int eachIteration = 50;
-        network.addListeners(new ScoreIterationListener(eachIteration));
+        CollectScoresIterationListener sc= new CollectScoresIterationListener();
```
Comment (MindTheSynapticGap) [inline src/main/java/Network.java]: Please change the name of this variable. Having names like "sc" in the code make it unreadable long-term.

File: src/main/java/NetworkConfiguration.java [hunk 3]
```diff
@@ -25,13 +25,11 @@ public NetworkConfiguration() {
                 .layer(0, new DenseLayer.Builder()
                         .nIn(NetworkProperties.INPUTS.property)
                         .nOut(NetworkProperties.HIDDEN_NODES.property)
-                        .activation(Activation.RELU)
```
Comment (MindTheSynapticGap) [inline src/main/java/NetworkConfiguration.java:28]: Should we have no activation function for layer 0 (and 1 for that matter)?

File: src/main/java/DataReader.java [hunk 4]
```diff
@@ -7,7 +7,7 @@
 import org.nd4j.linalg.dataset.SplitTestAndTrain;
 import org.nd4j.linalg.dataset.api.iterator.DataSetIterator;
 
-import java.io.IOException;
+import java.io.*;
```
Comment (MindTheSynapticGap) [inline src/main/java/DataReader.java]: Was this import necessary to change?

File: src/main/java/Network.java (hunk 2 above)
Comment (MindTheSynapticGap) [inline src/main/java/Network.java]: What's the purpose of CollectScoresIterationListener vs ScoreIterationListener?

Comment (s19487) [reply src/main/java/Network.java]: CollectScoresIterationListener has function exportScores(File f)

File: src/main/java/NetworkConfiguration.java (hunk 3 above)
Comment (s19487) [reply src/main/java/NetworkConfiguration.java:28]: We have the same activation function(SIGMOID) set for the whole network (line 20)

**Review by MindTheSynapticGap — APPROVED** [review]

**Pull request closed without merging**